### PR TITLE
LES-50 added a coinbase transaction context

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: les/coinbase-transaction-v1.jsonld [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+   "@context": {
+      "@version": 1.1,
+      "cbt": "https://sphereon-opensource.github.io/vc-contexts/les/coinbase-transaction-v1#",
+      "schema": "http://schema.org/",
+      "CoinbaseTransaction": {
```
Comment (nklomp) [inline les/coinbase-transaction-v1.jsonld]: I wouldn't call it CoinbaseTransaction. Simply CryptoTransaction is fine, as non of it really is coinbase specific

File: les/coinbase-transaction-v1.jsonld [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+   "@context": {
+      "@version": 1.1,
+      "cbt": "https://sphereon-opensource.github.io/vc-contexts/les/coinbase-transaction-v1#",
```
Comment (nklomp) [inline les/coinbase-transaction-v1.jsonld]: crypto-transaction and ct

File: les/coinbase-transaction-v1.jsonld [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      "schema": "http://schema.org/",
+      "CoinbaseTransaction": {
+         "@id": "https://sphereon-opensource.github.io/vc-contexts/les/coinbase-transaction-v1#rwc",
+         "@context": {
+            "@version": 1.1,
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "transactionId": "schema:String",
```
Comment (nklomp) [inline les/coinbase-transaction-v1.jsonld]: Since this is already about a transaction alone, I think we can leave out the transaction Prefixes.

**Review by nklomp — CHANGES_REQUESTED** [review]
Some small remarks

File: les/coinbase-transaction-v1.jsonld (hunk 1 above)
Comment (BtencateSphereon) [reply les/coinbase-transaction-v1.jsonld]: changed

File: les/coinbase-transaction-v1.jsonld (hunk 2 above)
Comment (BtencateSphereon) [reply les/coinbase-transaction-v1.jsonld]: changed

File: les/coinbase-transaction-v1.jsonld (hunk 3 above)
Comment (BtencateSphereon) [reply les/coinbase-transaction-v1.jsonld]: there is already an id field present. anyway we want to handle that differently or we want to keep the prefix for the transaction id?

Comment (nklomp) [reply les/coinbase-transaction-v1.jsonld]: Oh, hmzz, we probably can remove the first id=@id, given the txIds will be unique anyway

Comment (BtencateSphereon) [reply les/coinbase-transaction-v1.jsonld]: ok, changed

**Review by nklomp — APPROVED** [review]

**Pull request merged**